packages/cli/src/lib.rs

```diff
@@ -279,6 +286,44 @@ where
     // Render the template
     template.render(&target_dir, pkg_manager, &package_name, alpha, mobile)?;
 
+    // Install dependencies
+
+    if install_deps {
+        if pkg_manager.is_node() {
```

Comment:
I'd rather check if pkg_manager has an install command instead of checking for node only, keeps this condition true for future package managers that are not specific to node
```suggestion
    if install && pkg_manager.install_cmd().is_some() {
```